framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -767,6 +767,39 @@ public Range refineEqualTo(Range right) {
     }
 
     /**
+     * Refines this range to reflect that some value in it must not be equal to a value in the given
+     * range. This only changes the range if the given range (right) contains exactly one integer,
+     * and that integer is one of the bounds of this range. This is used for calculating the
+     * control-flow-refined result of the != operator. For example:
+     *
+     * <pre>
+     * <code>
+     *     {@literal @}IntRange(from = 0, to = 10) int a;
```

Comment:
Need to remove a leading space on this line and the next, to get alignment right in the generated Javadoc.  (Take a look!)